docs/open-data-model.md

```diff
@@ -0,0 +1,3310 @@
+    <td>string</td>
+    <td>Coming soon</td>
+    <td>Coming soon</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td>ssh_host_key_algorithm</td>
+    <td>string</td>
+    <td>Coming soon</td>
+    <td>Coming soon</td>
+  </tr>
+  <tr>
+    <td>DHCP</td>
+    <td>dhcp_assigned_ip4</td>
+    <td>bigint</td>
+    <td>Coming soon</td>
+    <td>Coming soon</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td>dhcp_mac</td>
+    <td>string</td>
+    <td>Coming soon</td>
+    <td>Coming soon</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td>dhcp_lease_time</td>
+    <td>double</td>
+    <td>Coming soon</td>
+    <td>Coming soon</td>
+  </tr>
+  <tr>
+    <td>IRC</td>
+    <td>irc_user</td>
+    <td>string</td>
+    <td>Coming soon</td>
+    <td>Coming soon</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td>irc_nickname</td>
+    <td>string</td>
+    <td>Coming soon</td>
+    <td>Coming soon</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td>irc_command</td>
+    <td>string</td>
+    <td>Coming soon</td>
+    <td>Coming soon</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td>irc_value</td>
+    <td>string</td>
+    <td>Coming soon</td>
+    <td>Coming soon</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td>irc_additional_data</td>
+    <td>string</td>
+    <td>Coming soon</td>
+    <td>Coming soon</td>
+  </tr>
+  <tr>
+    <td>Flow</td>
+    <td>flow_in_packets</td>
+    <td>int</td>
+    <td>Coming soon</td>
+    <td>Coming soon</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td>flow_out_packets</td>
+    <td>int</td>
+    <td>Coming soon</td>
+    <td>Coming soon</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td>flow_conn_state</td>
+    <td>string</td>
+    <td>Coming soon</td>
+    <td>Coming soon</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td>flow_history</td>
+    <td>string</td>
+    <td>Coming soon</td>
+    <td>Coming soon</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td>flow_src_dscp</td>
+    <td>string</td>
+    <td>Coming soon</td>
+    <td>Coming soon</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td>flow_dst_dscp</td>
+    <td>string</td>
+    <td>Coming soon</td>
+    <td>Coming soon</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td>flow_input</td>
+    <td>string</td>
+    <td>Coming soon</td>
+    <td>Coming soon</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td>flow_output</td>
+    <td>string</td>
+    <td>Coming soon</td>
+    <td>Coming soon</td>
+  </tr>
+  <tr>
+    <td>Vulnerability</td>
+    <td>vuln_id</td>
+    <td>string</td>
+    <td>Unique vulnerability identifier</td>
+    <td>10748</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td>vuln_type</td>
+    <td>string</td>
+    <td>Vulnerability title (i.e. Wireshark Multiple Vulnerabilities)</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td></td>
+    <td>vuln_status</td>
+    <td>string</td>
+    <td>Vulnerability type (Potential, Confirmed, etc.)</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td></td>
+    <td>vuln_severity</td>
+    <td>string</td>
+    <td>Vulnerability severity (Critical, High, etc.)</td>
+    <td></td>
+  </tr>
+  <tr>
```

Comment:
We could use event_time to capture this field if that's more efficient.

Reply:
ODM development is done in the topic branch https://github.com/apache/incubator-spot/tree/SPOT-181_ODM, please submit your PR to that branch...